bom/pom.xml

```diff
@@ -50,7 +50,7 @@
         <smallrye-open-api.version>1.1.1</smallrye-open-api.version>
         <smallrye-opentracing.version>1.3.0</smallrye-opentracing.version>
         <smallrye-fault-tolerance.version>2.0.4</smallrye-fault-tolerance.version>
-        <smallrye-jwt.version>1.1.3</smallrye-jwt.version>
+        <smallrye-jwt.version>1.1.4-SNAPSHOT</smallrye-jwt.version>
```

Comment:
Hmmm, is it a work in progress? Because we don't want a SNAPSHOT dependency there?

Other than that, I'll let @starksm64 review it.

Reply:
Ah, OK, I was too fast :).

Reply:
@gsmet np, yes, it is a work in progress, I added the label (coding in progress).

Reply:
I see, will add the [DO NOT MERGE] in the future